packages/neuron-ui/src/components/Receive/index.tsx

```diff
@@ -10,33 +11,73 @@ declare global {
   }
 }
 
+const AddressPanel = styled.div`
+  dispaly: flex;
+  display: -webkit-flex; /* Safari */
+  flex-direction: row;
+  margin: 10px 0 0 0;
+`
+
+const CopyImage = styled(Copy)`
+  width: 15px;
+  height: 25px;
+  padding-top: 10px;
+  margin-left: 10px;
+`
+
 const QRCodePanel = styled.div`
-  margin-top: 30px;
-  text-align: center;
+  width: 300px;
+  margin: 50px 0 0 30px;
 `
 
-const Address = styled.div`
-  color: black;
-  margin-top: 50px;
+const QRCodeModal = styled.div`
   text-align: center;
 `
 
-const Receive = () => {
+const Receive = ({ address }: { address: string }) => {
   const [t] = useTranslation()
-  const address: string = '0x0da2fe99fe549e082d4ed483c2e968a89ea8d11aabf5d79e5cbf06522de6e674'
+  const [showLargeQRCode, setShowLargeQRCode] = useState(false)
```

Comment:
component shouldn't have state, all state are just management in container, like MainContent

Reply:
You mean all states of components(belong to main content page) should be put in `containers/MainContent`?

Reply:
From a practice perspective, I think it's fairly fine to have simple 'state' like this within a pure component. This is only a toggle value to determine if the QR should be displayed in a large.

Reply:
A component with its inner state should be regarded as a widget.

There are three ways to display the larger one.

1. place the larger one in Dialog Widget, and set the dialog state in MainContent to toggle it.
2. add a new widget which has its own state to toggle the size of its picture.
3. use pure CSS to display/hide the image.

It's ok for now. Refactor it later.